nebula/ui/components/forms/VPNTextField.qml

```diff
@@ -10,50 +10,69 @@ import Mozilla.VPN 1.0
 import components 0.1
 
 TextField {
-    // TODO Add strings for Accessible.description, Accessible.name
     property bool hasError: false
     property bool showInteractionStates: true
     property bool forceBlurOnOutsidePress: true
     property alias _placeholderText: centeredPlaceholderText.text
+    property string _accessibleName: _placeholderText
+    property string _accessibleDescription: ""
 
     id: textField
 
+    Accessible.name: _accessibleName
+    Accessible.description: _accessibleDescription
+    Accessible.focused: textField.focus
```

Comment:
… does fix issue #4556 at least for VoiceOver on macOS in combination with setting `Accessible.focused`.